src/features/home/components/SeoFaq.tsx

```
@@ -0,0 +1,148 @@
+      answer:
+        'Superteam Earn is a platform where anyone can earn crypto by completing bounties, applying to projects, or securing grants from teams across the Solana ecosystem. Create a profile, showcase your work, and get rewarded for contributing to real products.',
+    },
+    {
+      question: 'How does Superteam Earn work?',
+      answer:
+        'Sponsors post opportunities such as bounties, projects, and grants. Anyone can apply or submit work, and rewards are distributed once the work is reviewed and approved. Everything happens through a single profile so your contributions build over time.',
+    },
+    {
+      question: 'Who is it for?',
+      answer:
+        'Anyone building in web3 — developers, designers, writers, researchers, community contributors, and operators — can participate. Whether you\'re new to crypto or already experienced, you can start contributing and earning.',
```

ESLint

[error] Replace `'Anyone·building·in·web3·—·developers,·designers,·writers,·researchers,·community·contributors,·and·operators·—·can·participate.·Whether·you\'re·new·to·crypto·or·already·experienced,·you·can·start·contributing·and·earning.'` with `"Anyone·building·in·web3·—·developers,·designers,·writers,·researchers,·community·contributors,·and·operators·—·can·participate.·Whether·you're·new·to·crypto·or·already·experienced,·you·can·start·contributing·and·earning."`

(prettier/prettier)